.github/workflows/link-check.yml

```diff
@@ -0,0 +1,17 @@
+name: Link Check
+
+on: [pull_request]
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: gaurav-nelson/github-action-markdown-link-check@1.0.12
+        with:
+          use-quiet-mode: "yes"
+          use-verbose-mode: "yes"
```

Comment:
Aren't these two settings a little contradictory? :)

The most detailed level of output seems to doable with quiet mode off, and verbose mode on. Well. :) Maybe we want to stick to the defaults here?

Reply:
for the record: https://github.com/gaurav-nelson/github-action-markdown-link-check#custom-variables

Reply:
those sound confusing indeed, this is my takeaway:

- quiet mode: don't show 2xx's
- verbose-mode: show http code for errors (404, 500, ...)